deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -191,6 +192,15 @@ spec:
           - '*'
           verbs:
           - '*'
+        # Permissions to access hostnetwork SCC
```

Comment:
Is this generated or did you manually add this?
The file should be generated from the deploy yamls

Reply:
Ahh sorry, I should've called it. The generation as of now is broken atleast for me. As of now, if you do `operator-sdk generate csv`, I am noticing that all the permissions are being taken out. I am using v0.18.1 of operator-sdk

`operator-sdk version: "v0.18.1", commit: "7bf7b6886d647dc202525daec16fab67dcc52a3d", kubernetes version: "v1.18.2", go version: "go1.13.6 linux/amd64"`

Reply:
Please bring this up with the operator SDK team and call it out in the README.

Reply:
if the generation doesnt work because of your changes thats a problem, if its unavoidable we need to change the readme to remove the generation

Reply:
I dont understand what you mean by `permissions are being take out` ?

Reply:
i tried running the command on my machine. I was able to generate the CSV with required permissions. Not sure why this is happening on your machine. 
i am on 
```
operator-sdk version: "v0.18.1", commit: "7bf7b6886d647dc202525daec16fab67dcc52a3d", kubernetes version: "v1.18.2", go version: "go1.13.6 linux/amd64"
```